src/ui/components/QuestionnaireEditForm.tsx

```diff
@@ -269,7 +274,9 @@ export const QuestionnaireEditForm = memo(
                 select
                 onChange={handleContextChange}
                 error={errorContextInput ? true : false}
-                value={questionnaire.context}
+                value={
+                  questionnaire.context.name ? questionnaire.context.name : ""
```

Comment:
`value={ questionnaire.context.name ?? ""}`
ça évite d'évaluer deux fois questionnaire.context.name

Reply:
didn't know the Nullish Coalescing operator, thanks :)